### ISSUE TITLE
Mysql vs MySQL

### ISSUE THREAD

**Issue by beliarh** [phalcon/cphalcon#11136]
https://github.com/phalcon/cphalcon/blob/master/phalcon/db/adapter/pdo/mysql.zep#L45
```zephir
class Mysql extends PdoAdapter implements AdapterInterface
```
https://github.com/phalcon/cphalcon/blob/master/phalcon/db/dialect/mysql.zep#L35
```zephir
class MySQL extends Dialect
```

**Comment by sergeyklay** [comment]
Can you please explain a bit more

**Comment by beliarh** [comment]
It would be nice if these classes had the same name

**Comment by sergeyklay** [comment]
Fixed in `2.1.x`
